src/files.js

```diff
@@ -192,6 +192,7 @@ FOAM_FILES([
   { name: "foam/web/URLState", flags: ['web'] },
   { name: "lib/input", flags: ['web'] },
   { name: "foam/box/Remote" },
+  { name: 'foam/audio/Beep' },
```

Comment:
Why do we need this?

Reply:
My bad, I was using it for debugging. It's easier to figure out what the wizard is doing if you can hear it.